docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -43,6 +43,21 @@ NOTE: The properties marked with *bold* are mandatory.
 |`PT1S`
 |The retry interval between subsequent requests
 
+|`http.auth.username`
+|character sequence
+|`<empty>`
+|The username, presence of this property requires the `http.auth.password` to be also set
+
+|`http.auth.password`
+|character sequence
+|`<empty>`
+|The password, presence of this property requires the `http.auth.username` to be also set
+
+|`http.auth.preemptive-auth-enabled`
+|`boolean`
+|`false`
+|See https://hc.apache.org/httpclient-legacy/authentication.html[Preemptive Authentication]
```

Comment:
too technical for end-users, please, extract important parts and put them as is without this link